.gitignore

```diff
@@ -31,11 +31,15 @@
 # Editors
 .vscode
 
-# build-in-source directory
-build/
```

Comment:
This is used for the library build.

Reply:
FYI, @samjwu we plan to move the documentation builds to a subfolder here in the future.

Reply:
I forgot about this; added it back in
[cleanup and fixes](https://github.com/ROCmSoftwarePlatform/hipSOLVER/pull/149/commits/5420c3d914738979c2dc5c97b6de7d1a317efe34)